client/model.go

```diff
@@ -923,20 +923,16 @@ func FieldMatchesFilter(path, value string) ListOpt {
 	}
 }
 
-// Sort enables sorting the policies by ascending order with given field.
-func Sort(field ListPoliciesSortingType) ListOpt {
-	return func(pf *policyListOptions) {
-		pf.sortingOptions = &sortingOptions{
-			field: field,
-		}
+// Sort enables sorting the policies by with given field.
+func Sort(field ListPoliciesSortingType, descending ...bool) ListOpt {
```

Comment:
This might be a little bit controversial but fits nicely as an optional field. Let me know if we should make this a mandatory argument.

Reply:
I prefer to avoid boolean arguments in public methods. I think there should be two `ListOpt`s here called `SortAscending` and `SortDescending`.